libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -980,6 +987,28 @@ bool RC_Channel::read_3pos_switch(RC_Channel::aux_switch_pos_t &ret) const
     return true;
 }
 
+// return switch position value as LOW, MIDDLE, HIGH
```

Comment:
This comment is rather incomplete considering the extra logic involved.  You need to explain that if reading the switch fails then it returns low.

Reply:
thank you, I fixed it.